src/index.js

```diff
@@ -14,4 +14,4 @@ ReactDOM.render(
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+
```

Comment:
This comment was a supplement to the `reportWebVitals()` function. Now, as it is removed, the comment should be removed as well